src/plugins/eos-external-apps-build-install

```diff
@@ -121,6 +121,11 @@ else
     exit 1
 fi
 
+# Fix permissions on export dir because some debs' tarballs base directory
+# has read permissions only for the user, and thus wouldn't be usable as a
+# Flatpak runtime
+chmod a+rx "$EXPORTDIR"
```

Comment:
Nitpick: you could omit `a`. But do you really also need execution permissions, in addition to read?